cpp/log/cert.cc

```diff
@@ -173,28 +173,35 @@ string ASN1ToStringAndCheckForNulls(ASN1_STRING* asn1_string,
 }
 
 
+unique_ptr<Cert> Cert::FromX509(ScopedX509 x509) {
+  return x509 ? unique_ptr<Cert>(new Cert(move(x509))) : nullptr;
+}
+
+
 Cert::Cert(ScopedX509 x509) : x509_(move(x509)) {
+  CHECK(x509_);
```

Comment:
I verified that I had caught all call sites of this by deleting this constructor entirely, and replacing the body of `Cert::FromX509` (the only expected call site, which avoids the `CHECK` by checking `x509` first) with `return nullptr;`, which compiled just fine.